libtiledbsoma/src/soma/managed_query.cc

```diff
@@ -942,7 +942,7 @@ bool ManagedQuery::_extend_and_write_enumeration(
     ArrowArray* value_array,
     ArrowSchema* index_schema,
     ArrowArray* index_array,
-    Enumeration& enmr,
+    Enumeration enmr,
```

Comment:
Below, I need to be returning this in a tuple: if the enumeration was extended I need to return a new `Enumeration` object; if not I need to return the existing one. Thus this existing one can no longer be a reference.